straxen/corrections_services.py

```diff
@@ -13,6 +13,8 @@
 corrections_w_file = ['mlp_model', 'gcn_model', 'cnn_model',
                       's2_xy_map', 's1_xy_map', 'fdc_map']
 
+single_value_corrections =['elife', 'baseline_samples_nv',
+                           'electron_drift_velocity']
```

Comment:
Perhaps we want to at some point query the database for this info

Reply:
sure, we can do that in the future